### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by 3NCRY9T3R**

### CODE REVIEW

**Review by kpiyush04 — COMMENTED** [review]
update readme with more description.

File: Hacktoberfest/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        ![](https://i.ibb.co/1RYZWbm/Capture.png)
+
+  **7.** On the next window click on the **Create Pull Request** button.
+        ![](https://i.ibb.co/VCLckB3/Capture.png)
+
+**You will have to repeat the steps 4 to 7 for 4 times with different programs in order to be eligible for a free T-shirt and stickers
+
+**Rules**
+**1.** Don't copy the code from anywhere on the net.
+**2.** Beginner's level code like 'helloworld', 'sum of 2 nos.', etc.
```
Comment (shreyanshdeep) [inline Hacktoberfest/README.md:30]: Please clear this point

**Comment by shreyanshdeep** [comment]
Add more details and if possible few images too.

**Review by aanurraj — COMMENTED** [review]
Is this legal ??

**Comment by ravisaxena23** [comment]
@3NCRY9T3R Please add these things too.
1. basic git commands.
2. description why we need github.
3. har repo jispr jo b pr krni h tshirt ke lie vo step by step add kro ki phele yaha jao fir yaha jao ye kro * 5 times krna blhi hi same steps repeat ho.
4. how to fork(why we should fork) ye add kro.
5.  apna no htao tj ka mail dalo last mai.
6. sirf tj ke repo use kro.
7. ** ke jgha # use kr skte ho.
8. tohda sa systematic kro esa socho tm sbse phele kya krte agr audience hote. like phele github account fir event regestration fir basic explanition etc include 2nd year too tmko kaam asan pdega.

**Comment by ravisaxena23** [comment]
@aanurraj its legal.

**Comment by aanurraj** [comment]
> @aanurraj its legal.
> 

Okay!

**Review by aanurraj — APPROVED** [review]

**Review by Isha2103 — COMMENTED** [review]
Fine👍

**Pull request closed without merging**